### DISCUSSION TITLE
Funktioniert FreetzNG mit einer 7530 AX?

### DISCUSSION THREAD

**Discussion by nixmomo** [Freetz-NG/freetz-ng#619] [Allgemein] [answered]
Hi,
mich würde interessieren ob Freetz NG mit der 7530 AX funktioniert. 
Alles was ich dazu finde ist über ein Jahr alt, laut Liste wird sie theoretisch Supported aber lesen tue ich in verschiedenen Threads eben was anderes.

Wird es also unterstützt und funktioniert es mit den Tut's die verlinkt sind?
https://freetz-ng.github.io/freetz-ng/wiki/10_Beginner/install.de.html

**Comment by freetz-ng-mod** [comment, marked as answer]
Bilder sagen mehr als 1000 Worte
![7530ax](https://user-images.githubusercontent.com/79326105/192577954-27d594a9-ec29-4ba7-b6f2-4de77cce4db5.png)

**Comment by nixmomo** [comment]
Und ganz normal über die Anleitung die verlinkt war? Ich hab da ständig was gelesen das die speziell geflasht werden wollen?

**Comment by freetz-ng-mod** [comment]
per `./push_firmware IMAGE_NAME -cmd ftp`
Mit Windows geht das nicht

**Reply by nixmomo** [reply]
Das es per Windoof nicht geht ist mir klar, zum builden muss ich ja eh eine Freetz VM aufsetzen. In dieser dann einfach den Befehl ? Das push_firmware ist dann im git welches geclont wurde nehm ich an?

Sry für die dummen Fragen aber ich hab wenig Lust auf Recovery :D

**Comment by nixmomo** [comment]
Jetzt muss ich leider noch mal Fragen denn in der Doku finde ich dazu nichts.... zudem ist auch diese leider stellenweise total veraltet ...

```
freetz@freetz-linux:~/freetz-ng$ ./push_firmware latest.image -cmd ftp -ip 192.168.2.1
-bash: ./push_firmware: No such file or directory
freetz@freetz-linux:~/freetz-ng$ cd tools
freetz@freetz-linux:~/freetz-ng/tools$ ./push_firmware latest.image -cmd ftp -ip 192.168.2.1

 * Analyzing 'latest.image' ...
 * Product: Fritz_Box_HW256 (FRITZ!Box 7530 AX)
 * Using command: ftp
 * Target host: 192.168.2.1
 * Outgoing IP: 192.168.2.22
 * Flash mode: fit-boot
 * Allowed memory size: 512 MB
 * Designated linux_fs_start: <other>

 !!! WARNING !!! WARNING !!! WARNING !!! WARNING !!! WARNING !!!
 !!!  THERE IS NO WARRANTY AT ALL !!! USE AT YOUR OWN RISK   !!!

 * Are you sure, that you want to flash this file to the device?
   latest.image
   Proceed? (y/[n]) y

 * You should now reboot your box (192.168.2.1). Waiting for shut down.
   Switch off, if reboot is not detected because it happens too quickly.
   Some newer bootloader versions allow to flash on power-cycle only.
   ...............................
 * MAPSTART=0x00000000
 * FULLSIZE=0x20000000   (512 MB)
 * MAPLIMIT=0x20000000
 * FILESIZE=0x0251461b   (~37 MB)
 * ALIBYTES=0x0000b9e5   (~46 kB)
 * FREESIZE=0x1dae0000   (~474 MB)
 * MTDSTART=0x1dae0000

 * No reply from box, assuming switch-off or restart. Trying to re-detect box.
   Waiting ................................................................. found!

 * Box is back up again, initiating transfer.

ftp: Can't connect to `192.168.2.1:21': Connection refused
ftp: Can't connect to `192.168.2.1:ftp'
Not connected.
Debugging on (debug=1).
Not connected.
Not connected.
Not connected.
Not connected.
Not connected.

done
freetz@freetz-linux:~/freetz-ng/tools$
```
Er Sagt zwar "done" aber von Freetz sehe ich nix und unter Port 81 kommt auch nur ne refused Meldung.... Ich sehe hier auch kein Flashvorgang.... Ich hab einfach den Befehl ausgeführt, die Box vom Strom genommen und wieder eingesteckt .... Muss hier noch irgendwas anderes beachtet werden?? 

Wieso findet man dazu nix in der Doku??

**Comment by freetz-ng-mod** [comment]
./push_firmware latest.image -cmd ftp 
mach es mal so

[Da kannst auch sehen wie es ausschaut](https://github.com/Freetz-NG/freetz-ng/discussions/483)

**Comment by nixmomo** [comment]
Dann meckert er rum das er unter 192.168.178.1 keine box findet was ja logisch ist denn ich hab hier ne andere IP Struktur und deswegen auch die IP mit angegeben

**Comment by freetz-ng-mod** [comment]
Hat dein Linux eine feste IP. Wenn nicht mach das mal

**Comment by nixmomo** [comment]
das ist das freetzlinux also ja ich gehe davon aus? Müsste ich nachher mal schauen

**Comment by freetz-ng-mod** [comment]
- IP-Adresse des Rechners auf "192.168.178.32" stellen. Gateway auf "192.168.178.1", Subnet auf 255.255.255.0

**Comment by nixmomo** [comment]
dann muss ich die IP der Fritte vorher wieder auf 192.168.178.1 setzen right?

**Comment by freetz-ng-mod** [comment]
nein warum denn.

**Comment by nixmomo** [comment]
weil das gateway gar nicht existiert? Die Box hat die ip 192.168.2.1 und da werd ich mit nem gw 192.168.178.1 nicht weit kommen?

**Comment by freetz-ng-mod** [comment]
Die Box wird doch vorher schon angehalten.ist wie beim recover.exe

**Comment by nixmomo** [comment]
aber er prüft doch ob die ip erreichbar ist und wartet dann auf den reboot aber gut also die ip vom pc auf irgendwas ala 192.168.178.100 mit gw 192.168.178.1 und die vm dann auf die ip 192.168.178.32 mit gw 192.168.178.1 

right?

**Comment by nixmomo** [comment]
ok also wieso auch immer.... Fritzbox ist da eventuell komisch.... Ich hab die IP in der VM mit 
`ifconfig eth0 192.168.178.32 netmask 255.255.255.0 broadcast 192.168.178.255`
geändert und im Windoof auch statisch eine 192.168.178.111 mit Gateway 192.168.178.1 vergeben.
So kam ich zumindest auf die Virtualbox Kiste wieder drauf und dann hab ich tatsächlich mittels 
./push_firmware latest.image -cmd ftp 
Freetz flashen können.... Und das obwohl die Box als IP ne ganz andere hat.
Das der FTP Dienst beim Booten immer auf die 192.168.178.1 lauscht war mir unklar an der Stelle.... So ging es zumindest und ich hab endlich wieder mein OpenVPN Client und mein dnsmasq für die internen Domains.... was ein Krampf aber am Ende dauerte es nur paar Minuten und ging fehlerfrei.

Danke!!!
